packages/notification/channel/teams.ts

```diff
@@ -26,7 +26,7 @@ import Joi from 'joi'
 import { sendHttpRequest } from '../utils/http'
 import type { NotificationMessage } from '.'
 
-type NotificationData = {
+export type NotificationData = {
```

Comment:
Could you elaborate why it needs to be exported?

Reply:
since `sendWithCustomContent(notificationData, content)` will be used by neosene, then the type of the parameter need to be imported there to be exactly have same type